packages/suite/src/actions/suite/__tests__/analyticsActions.test.ts

```diff
@@ -1,146 +0,0 @@
-import { configureStore } from '@suite/support/tests/configureStore';
-
-import analyticsReducer from '@suite-reducers/analyticsReducer';
-import * as analyticsActions from '@suite-actions/analyticsActions';
-import { ANALYTICS } from '../constants';
-
-type AnalyticsState = ReturnType<typeof analyticsReducer>;
-
-interface InitialState {
-    analytics?: Partial<AnalyticsState>;
-}
-
-jest.mock('@suite-utils/random', () => ({
-    __esModule: true,
-    getTrackingRandomId: () => 'very-random',
-}));
-
-export const getInitialState = (state: InitialState | undefined) => {
-    const analytics = state ? state.analytics : undefined;
-    return {
-        suite: {
-            tor: false,
-            flags: { initialRun: false },
-        },
-        analytics: {
-            ...analyticsReducer(undefined, { type: 'foo' } as any),
-            ...analytics,
-        },
-    };
-};
-
-type State = ReturnType<typeof getInitialState>;
-const mockStore = configureStore<State, any>();
-
-const initStore = (state: State) => {
-    const store = mockStore(state);
-    store.subscribe(() => {
-        const action = store.getActions().pop();
-        const { analytics } = store.getState();
-        store.getState().analytics = analyticsReducer(analytics, action);
-        // add action back to stack
-        store.getActions().push(action);
-    });
-    return store;
-};
-
-describe('Analytics Actions', () => {
-    beforeAll(() => {
-        jest.spyOn(console, 'error').mockImplementation();
-    });
-    afterAll(() => {
-        jest.clearAllMocks();
-    });
-
-    it('analyticsActions.init() - unconfirmed', () => {
-        const state = getInitialState({});
-        const store = initStore(state);
-        store.dispatch(
-            analyticsActions.init({
-                enabled: undefined,
-                confirmed: false,
-                sessionId: undefined,
-                instanceId: undefined,
-            }),
-        );
-        store.dispatch(
-            analyticsActions.init({
-                enabled: false,
-                confirmed: false,
-                sessionId: undefined,
-                instanceId: undefined,
-            }),
-        );
-        store.dispatch(
-            analyticsActions.init({
-                enabled: true,
-                confirmed: false,
-                sessionId: undefined,
-                instanceId: undefined,
-            }),
-        );
-
-        const unconfirmedInitAction = {
-            type: ANALYTICS.INIT,
-            payload: { enabled: true, sessionId: 'very-random', instanceId: 'very-random' },
-        };
-
-        expect(store.getActions()).toMatchObject([
-            unconfirmedInitAction,
-            unconfirmedInitAction,
-            unconfirmedInitAction,
-        ]);
-    });
-
-    it('analyticsActions.init() - enabled confirmed', () => {
```

Comment:
is this removed without any replacement? the unconfirmed logic is kinda crucial for us. We can lose critical data

Reply:
https://github.com/trezor/trezor-suite/pull/7622/commits/9699a35bef2f083ee0118dad7e93d3a7474def8b

Reply:
I thought we won't be testing slice reducers as this is not as complicated on logic as typescript checks whether types are correct and then it's just kind of "documentation" to have these tests. Since the API changed and I couldn't just move these tests to common, I figured I'll get rid of this. 

But if you think it should be tested as a critical part of app, I can put them back 😉

Reply:
https://github.com/trezor/trezor-suite/pull/7622/commits/16bc07eb8cf54f4b869d6e7028c4693fd3d63e4a